packages/frontend/app/lib/apollo.server.ts

```diff
@@ -16,8 +13,8 @@ BigInt.prototype.toJSON = function (this: bigint) {
   return this.toString()
 }
 
-if (!global.__apolloClient) {
-  global.__apolloClient = new ApolloClient({
+export function getApolloClient(token?: string) {
```

Comment:
We need to re-instantiate the Apollo Client each time a request is made now, so that it is able to pick up whether or not an authorization token is present.